--- ptlib-2.10.0/include/ptlib/object.h.orig	2011-02-23 05:53:19.000000000 +0100
+++ ptlib-2.10.0/include/ptlib/object.h	2011-04-28 13:27:31.000000000 +0200
@@ -977,7 +977,7 @@
 
 
 // Memory pooling allocators
-#if defined(__GNUC__) && __GNUC__ >= 4 && !defined(P_MINGW) && !defined(P_MACOSX)
+#if 0
 #include <ext/bitmap_allocator.h>
 template <class Type> class PFixedPoolAllocator    : public __gnu_cxx::bitmap_allocator<Type> { };
 #include <ext/mt_allocator.h>
