--- ptlib-2.10.11/ptlib.m4.orig	2013-08-15 01:20:33.000000000 +0200
+++ ptlib-2.10.11/ptlib.m4	2015-08-01 22:00:11.565907729 +0200
@@ -472,17 +472,19 @@
 
           AC_CHECK_HEADERS([lua.h], [ptlib_lua=yes], [ptlib_lua=no])
           if test "x${ptlib_lua}" = "xyes" ; then
-            AC_CHECK_LIB([lua], [lua_newstate], [ptlib_lua=yes], [ptlib_lua=no])
+            AC_CHECK_LIB([lua5.2], [lua_newstate], [ptlib_lua=lua5.2], [
+            AC_CHECK_LIB([lua51], [lua_newstate], [ptlib_lua=lua51], [
+            AC_CHECK_LIB([lua], [lua_newstate], [ptlib_lua=lua], [ptlib_lua=no])])])
           fi
 
           LIBS="$old_LIBS"
           CPPFLAGS="$old_CPPFLAGS"
 
-          if test "x${ptlib_lua}" = "xyes" ; then
-            LUA_LIBS="-llua ${ptlib_lua_libs}"
+          if test "x${ptlib_lua}" != "xno" ; then
+            LUA_LIBS="-l${ptlib_lua} ${ptlib_lua_libs}"
             LUA_CFLAGS="${ptlib_lua_cflags}"
           fi
-          AS_IF([test AS_VAR_GET([ptlib_lua]) = yes], [$1], [$2])[]
+          AS_IF([test AS_VAR_GET([ptlib_lua]) != no], [$1], [$2])[]
          ])
 
 
